### ISSUE TITLE
rectangular size input on transfer learning

### ISSUE THREAD

**Issue by chris-tkinter** [facebookresearch/TimeSformer#60]
Hi, thanks for the awesome model + pretrained weights!

I am working on a video classification task using the pretrained SSv2 8-224 model. I loaded the pretrained weights, unfreezed the whole model, and used it on my downstream videos to get the class logits. However the result wasn't good enough and I would like to improve further.

My videos are of size (720, 1280), so to use SSv2 8-224 model currently I resized those video frames into 224x224. I suspected that this would lead to the loss of some crucial information and I am wondering if this model is able to take an non-square frames as an input.

Thanks!

**Comment by gberta** [comment]
You might need to make a few minor modification in the data loading, and model definition files for this. In theory,  using non-square inputs frames should work as well.
